json-csharp/HowToSerializeAListToJsonInCSharp/Tests/NewtonsoftJsonMethodsTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using HowToSerializeAListToJsonInCSharp;
+
+namespace Tests
+{
+    public class NewtonsoftJsonMethodsTests
+    {
+        private static readonly List<Club> _englishClubs = new()
+            {
+                new Club
+                {
+                    Name = "Arsenal",
+                    YearFounded = 1886,
+                    Country = "England",
+                    NumberOfPlayers = "26",
+                },
+                new Club
+                {
+                    Name = "Manchester City",
+                    YearFounded = 1880,
+                    Country = "England",
+                    NumberOfPlayers = "25",
+                },
+                new Club
+                {
+                    Name = "Sunderland",
+                    YearFounded = 1879,
+                    Country = "England",
+                    NumberOfPlayers = "30",
+                }
+            };
+
+        private readonly SerializeListToJsonWithNewtonsoftJson _serializeList
+            = new(_englishClubs);
+
+        private readonly string _expectedResult
+            = @"[
+  {
+    ""name"": ""Arsenal"",
+    ""yearFounded"": 1886,
+    ""country"": ""England"",
+    ""numberOfPlayers"": ""26""
+  },
+  {
+    ""name"": ""Manchester City"",
+    ""yearFounded"": 1880,
+    ""country"": ""England"",
+    ""numberOfPlayers"": ""25""
+  },
+  {
+    ""name"": ""Sunderland"",
+    ""yearFounded"": 1879,
+    ""country"": ""England"",
+    ""numberOfPlayers"": ""30""
+  }
+]";
```

Comment:
Would "[raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/reference-types#string-literals)" from C# 11 help here so you don't have to indent the text all the way to the left?

Play around a bit and let me know. This might help you professionally as well, and I'm just curious :)